### PR TITLE
Country Page - fetch country resources & expert, code refactor, tests

### PR DESCRIPTION
**Pull request by davidbgomes**
[Turn static country page into dynamic one: resources and expert sections](https://app.clickup.com/t/865chqecc)

- created the country page test and added some base tests;
- fetched country resources, with new hook;
- fetched country experts, with new hook;
- Refactored some rushed code.

### CODE REVIEW

**Review by xurxodev — CHANGES_REQUESTED** [review]
thanks @davidbgomes 

Great work in this PR, congratulations!

I've found only an issue to fix and I've added it in comments

**Comment by davidbgomes** [comment]
Thank you @xurxodev! The pair programming helped a lot to get started with the tests. I've corrected the Person type as well, should be good now. Thanks!

File: src/domain/entities/Person.ts [hunk 1]
```diff
@@ -5,4 +5,5 @@ export type Person = {
     position: string;
     institution?: string;
     countryId?: string;
+    countryName?: string;
```
[the remark this answers is not in the document]
Comment (davidbgomes) [reply src/domain/entities/Person.ts]: thanks for the explanation, I must have added for a type error and never corrected it.

**Review by xurxodev — CHANGES_REQUESTED** [review]
thanks @davidbgomes 

The Typescript check is failing in the last commit.

You can review executing locally:
yarn tsc

**Comment by davidbgomes** [comment]
Thank you @xurxodev , sorry for the delay.. missed the notification for your review comment. Should be good now!

**Review by xurxodev — APPROVED** [review]
thanks @davidbgomes

**Pull request merged**